labman/db/tests/integration_tests.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python
+
+from datetime import datetime
+from itertools import chain
+
+import numpy as np
+import click
+
+from labman.db.process import (
+    SamplePlatingProcess, GDNAExtractionProcess, GDNAPlateCompressionProcess,
+    LibraryPrep16SProcess, NormalizationProcess, QuantificationProcess,
+    LibraryPrepShotgunProcess, PoolingProcess, SequencingProcess)
+from labman.db.user import User
+from labman.db.plate import PlateConfiguration, Plate
+from labman.db.equipment import Equipment
+from labman.db.composition import ReagentComposition
+from labman.db.sql_connection import TRN
+
+
+def get_samples():
```

Comment:
Is this... *all* of the samples in the qiita db?

Reply:
Yes, even on the test environment (with more than 500k samples), it takes 1-2 seconds to run.